beacon-chain/beacon-network.md

```diff
@@ -204,12 +204,12 @@ content_key                          = selector + SSZ.serialize(light_client_opt
 ```
 
 > The `LightClientOptimisticUpdate` objects are ephemeral and only the latest is
-of use to the node. The content key requires the `optimistic_slot` to be
-provided so that this object can be more efficiently gossiped. Nodes should
-decide to reject an `LightClientOptimisticUpdate` in case it is not newer than
-the one they already have.
-For `FindContent` requests, a node should know the current slot as it is time
-based.
+of use to the node. The content key requires the `optimistic_slot` (corresponding to
+the `slot` in the `attested_header` contained in the update to be provided so that this
+object can be more efficiently gossiped. Nodes should decide to reject an
+`LightClientOptimisticUpdate` in case it is not newer than the one they already have.
+For `FindContent` requests, a node should compute the current slot based on its local clock
+and then use that slot minus 1 as a starting point for retrieving the most recent update.
```

Comment:
Does it make sense to add? "If none is found in the previous slot, then the node can search slot by slot backwards through time, or just wait to receive an offer for a new slot"

Reply:
How about, "If none is found in the previous slot, then the node should preferably wait for an update to be gossiped rather than flood the network with requests for stale updates."

I think my concern is if we suggest people start asking for stale data on a network that should be continuously updating this via gossip, we're just going to DoS ourselves.